src/Common/ErrorCodes.cpp

```diff
@@ -603,8 +603,11 @@
     M(632, UNEXPECTED_DATA_AFTER_PARSED_VALUE) \
     M(633, QUERY_IS_NOT_SUPPORTED_IN_WINDOW_VIEW) \
     M(634, MONGODB_ERROR) \
-    M(635, CANNOT_POLL) \
     M(636, CANNOT_EXTRACT_TABLE_STRUCTURE) \
+    M(650, SNAPPY_UNCOMPRESS_FAILED) \
+    M(651, SNAPPY_COMPRESS_FAILED) \
+    M(652, NO_HIVEMETASTORE) \
+    M(635, CANNOT_POLL) \
```

Comment:
Please return correct order of error codes, it must be ascending.

Reply:
done